infra/resources/prod/function_ioweb_profile.tf

```
@@ -0,0 +1,300 @@
+      // 1 hour
+      EXCHANGE_JWT_TTL                   = "3600"
+      MAGIC_LINK_JWE_PRIMARY_PRIVATE_KEY = data.azurerm_key_vault_secret.magic_link_jwe_private_key.value
+      MAGIC_LINK_JWE_PRIMARY_PUB_KEY     = data.azurerm_key_vault_secret.magic_link_jwe_pub_key.value
+      MAGIC_LINK_JWE_ISSUER              = local.function_JWT_issuer
+      MAGIC_LINK_BASE_URL                = "https://ioapp.it/it/blocco-accesso/magic-link/"
+      // TBD: more/less than 1 week?
+      MAGIC_LINK_JWE_TTL = "604800"
+
+      HUB_SPID_LOGIN_JWT_ISSUER  = "api-web.io.pagopa.it/ioweb/auth"
+      HUB_SPID_LOGIN_JWT_PUB_KEY = data.azurerm_key_vault_secret.spid_login_jwt_pub_key.value
+
+      // -------------------------
+      // Fast Login config
+      // -------------------------
+      FAST_LOGIN_API_KEY         = data.azurerm_key_vault_secret.functions_fast_login_api_key.value
+      FAST_LOGIN_CLIENT_BASE_URL = "https://io-p-itn-auth-lv-func-02.azurewebsites.net"
+
+      // -------------------------
+      // Functions App config
+      // -------------------------
+      FUNCTIONS_APP_API_KEY         = data.azurerm_key_vault_secret.functions_app_api_key.value
+      FUNCTIONS_APP_CLIENT_BASE_URL = "https://io-p-itn-auth-profile-fn-02.azurewebsites.net"
+
+      // -------------------------
+      // Hub Spid Login for ioweb config
+      // -------------------------
+      HUB_SPID_LOGIN_API_KEY         = data.azurerm_key_vault_secret.spid_login_api_key.value
+      HUB_SPID_LOGIN_CLIENT_BASE_URL = "https://io-p-weu-ioweb-spid-login.azurewebsites.net"
+
+      // -------------------------
+      // Audit Logs config
+      // -------------------------
+      AUDIT_LOG_CONNECTION_STRING = data.azurerm_storage_account.immutable_spid_logs_storage.primary_connection_string
+      AUDIT_LOG_CONTAINER         = local.immutable_audit_logs_container_name
+    }
+  }
+}
+
+resource "azurerm_resource_group" "function_web_profile_rg" {
+  name     = "${local.project}-${local.domain}-${local.function_ioweb_profile.name}-rg-01"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "function_web_profile" {
+  source = "github.com/pagopa/dx//infra/modules/azure_function_app?ref=ab26f57ed34a614fd3fa496c7b521be9ecc88e1b"
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    domain          = local.domain
+    app_name        = local.function_ioweb_profile.name
+    instance_number = "01"
+  }
+
+  resource_group_name = azurerm_resource_group.function_web_profile_rg.name
+  health_check_path   = "/api/v1/info"
+  node_version        = 20
+  app_service_plan_id = data.azurerm_app_service_plan.shared_plan_itn.id
+
+  subnet_cidr   = local.cidr_subnet_fn_web_profile
+  subnet_pep_id = data.azurerm_subnet.private_endpoints_subnet.id
+
+  subnet_service_endpoints = {
+    web     = true,
+    storage = true
```

```suggestion
    storage = true
```
since we can connect via private endpoint, do you believe `Microsoft.Web` service endpoint can be disabled for this function?